.cirrus.yml

```diff
@@ -18,7 +18,7 @@ task:
     fingerprint_script: echo $OS; grep -r --include=pubspec.yaml 'PUBSPEC CHECKSUM' "$CIRRUS_WORKING_DIR"
   flutter_pkg_cache:
     folder: bin/cache/pkg
-    fingerprint_script: echo $OS; cat bin/internal/engine.version
+    fingerprint_script: echo $OS; cat bin/internal/*.version
```

Comment:
I think you missed it for other platforms like Windows and macOS

Reply:
Ah good point. Goldens are currently only tested on Linux, but we should keep it consistent regardless. Thanks!